marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/main.sjs

```diff
@@ -16,7 +16,7 @@ const traceEvent = consts.TRACE_MAPPING_DEBUG;
 function beforeMain(contentSequence, stepExecutionContext) {
   const path = stepExecutionContext.flowStep.options.mappingParametersModulePath;
   if (path) {
-    const values = require(path)["getParameterValues"](contentSequence, stepExecutionContext);
+    const values = require(path)["getParameterValues"](contentSequence);
```

Comment:
There should be no "beforeMain" right ? Should there be changes to this file and lib.sjs in this PR ? I will wait until #5594 is merged before I test this out.

Reply:
Rebased, beforeMain is gone now.